### ISSUE TITLE
tvOS accessible Names and descriptions

### ISSUE THREAD

**Issue by mulundapm** [bcc-code/bcc-media-tvos#4]
"After the new update to AppleTV, accessibility voiceovers are limited and very generic. The descriptions just say “button” without any description. we have friends in Salem that rely on accessibility options and would like to see if it can be improved. Thank you!"

**Comment by fredrikvedvik** [comment]
Ready in test flight. Can be released with the next version.

**Comment by mulundapm** [comment]
For future changes, would be very helpful to link the pull requests so that we could trace back to which version include the fix

**Comment by mulundapm** [comment]
Is this released?

**Comment by fredrikvedvik** [comment]
Accessibility improvements should be released with 4.0.6

4.0.7/4.0.8 is stuck in app review.
